x-pack/heartbeat/monitors/browser/synthexec/enrich.go

```diff
@@ -206,6 +205,9 @@ func (je *journeyEnricher) createSummary(event *beat.Event) error {
 		return je.error
 	}
 
+	// create a new check group for the next journey
+	je.streamEnricher.checkGroup = makeUuid()
```

Comment:
Question: Would check-group be unique if we run zip urls which could contain multiple journeys since we moved the generation to the stream enricher?

Reply:
This code should still create multiple check groups for zip URLs. I don't think we want to assert this at the new functional tests though, since my hope is that zip urls will be gone soon enough.